### ISSUE TITLE
Test for encrypted OSDs

### ISSUE THREAD

**Issue by smithfarm** [SUSE/sesdev#251]
When `--encrypted-osds --qa-test` is given, the QA test should assert that the OSDs are, indeed, encrypted.

**Comment by smithfarm** [comment]
blocked by #241

**Comment by smithfarm** [comment]
http://smithfarm-thebrain.blogspot.com/2020/03/how-to-verify-that-encrypted-osd-is.html

**Comment by asettle** [comment]
Out of scope
